### DISCUSSION TITLE
Showing the window title when matching commands?

### DISCUSSION THREAD

**Discussion by gabyx** [hluk/CopyQ#2536] [Q&A]
How can I find out the window name copyq is matching when copying, to check if my regex actually works?
Because .*warden for the bitwarden app on linux does not work apparently....

**Comment by gabyx** [comment]
I am under wayland, so it apparently does not work?

**Reply by laur89** [reply]
[this reply](https://github.com/hluk/CopyQ/discussions/2172#discussioncomment-4132705) suggests that is the case - no way to access window titles under wayland.

**Comment by hluk** [comment]
Depending on the window manager (or desktop environment), there are external commands that can retrieve the window title.

The [Wayland Support](https://github.com/hluk/copyq-commands/tree/master/Scripts#wayland-support) command supports getting window title in KDE, Sway and Hyprland ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)).
